google/cloud/apigee/registry/v1/registry_models.proto

```diff
@@ -59,7 +59,7 @@ message Project {
 message Api {
   option (google.api.resource) = {
     type: "registry.googleapis.com/Api"
-    pattern: "projects/{project}/apis/{api}"
+    pattern: "projects/{project}/locations/{location}/apis/{api}"
```

Comment:
I think it might be a breaking change (for client libraries) to switch from "global" to "{location}" (and I think that eventually we'll want to support non-global locations)

Reply:
Makes sense. Just so that I understand it correctly, the core registry server will be able to serve requests like CreateApi under parent `projects/xyz/locations/us-central1`. But these requests should be blocked at the One Platform API level if the location is anything other than `global`(until we support the configuration), correct?

Reply:
I'm thinking that on startup, the registry server would read a configuration that sets the "location" value (defaulting to "global"). This would be the location where the server is running and would be the only location that the server would recognize in requests or return in responses. Thus if the location was "us-central1", then "global" wouldn't work, and vice-versa.

Reply:
Thanks for clarifying!